src/git.zsh

```diff
@@ -245,6 +335,71 @@ _fzf_complete_git-unstaged-files_post() {
     done
 }
 
+_fzf_complete_git-remotes() {
+    local fzf_options=$1
+    shift
+
+    _fzf_complete "--ansi --tiebreak=index $fzf_options" $@ < <(git remote --verbose 2> /dev/null | awk '
+        /\(fetch\)$/ {
+            gsub("\t", " ")
+            print
+        }
+    ' | _fzf_complete_git_tabularize)
+}
+
+_fzf_complete_git-remotes_post() {
+    awk '{ print $1 }'
+}
+
+_fzf_complete_git-refs() {
+    local fzf_options=$1
+    shift
+
+    local ref=${${$(git config remote.$repository.fetch 2> /dev/null)#*:}%\*}
+
+    [[ -z $ref ]] && return
```

Comment:
Put this into `if` clause.

Reply:
This line is added because `git for-each-ref $ref` shows all refs when `$ref` is empty. I delete it because it is solved by quoting `$ref`.

Reply:
OK